app/views/devise/registrations/start.html.erb

```diff
@@ -7,29 +7,52 @@
         <%= render "devise/shared/error_messages", resource: resource, resource_error_messages: @resource_error_messages %>
       <% end %>
 
+      <%= render "govuk_publishing_components/components/heading", {
+        text: yield(:title),
+        heading_level: 1,
+        font_size: "xl",
+        margin_top: 0,
+        margin_bottom: 3,
+      } %>
+
+      <% if params[:jwt] %>
+        <%= hidden_field_tag :jwt, params[:jwt] %>
+      <% end %>
+
       <%= render "govuk_publishing_components/components/input", {
         label: {
-          text: yield(:title),
+          text: t("devise.registrations.start.fields.email.label"),
+        },
+        name: "user[email]",
+        type: "email",
+        id: "email",
+        value: @email,
+        error_message: devise_error_items(:email, @resource_error_messages),
+        autocomplete: "email",
```

Comment:
Consider: using `username` over `email` for better support with password managers - detail in main comment
```suggestion
        autocomplete: "username",
```